src/poetry/packages/locker.py

```diff
@@ -49,7 +49,11 @@
 
 logger = logging.getLogger(__name__)
 _GENERATED_IDENTIFIER = "@" + "generated"
-GENERATED_COMMENT = f"This file is automatically {_GENERATED_IDENTIFIER} by Poetry and should not be changed by hand."
+GENERATED_COMMENT = (
+    f"This file is automatically {_GENERATED_IDENTIFIER} by Poetry and should not be"
+    " changed by hand."
+)
```

Comment:
this might be overkill. You can probably hard code this (in one place) as a single string. Certainly it's not necessary to construct the identifier from two strings

```python
GENERATED_COMMENT = (
    f"This file is automatically @generated by Poetry and should not be"
    " changed by hand."
)
```

Reply:
In my experience, you don't want the `@generated` string in your source since it causes tooling to not show changes to the source file itself

Reply:
ah gotcha